pkg/util/utils.go

```diff
@@ -43,6 +39,15 @@ func GetPlatform(platform string, allPlatforms []sippyprocessingv1.PlatformResul
 	return nil
 }
 
+func FindPrevBugzillaJobFailures(bzComponent string, bugzillaJobFailures []sippyprocessingv1.SortedBugzillaComponentResult) *sippyprocessingv1.SortedBugzillaComponentResult {
```

Comment:
not really specific to Prev, right?  is this just "FindBugzillaJobFailures"?  FindJobFailuresForBZComponent?

Reply:
> not really specific to Prev, right? is this just "FindBugzillaJobFailures"? FindJobFailuresForBZComponent?

Yeah, rough name from long before.  Followup?

Reply:
i'm aware the legacy is mine, just figured if you're gonna rename it, let's get it right :)

followup is fine.